e2e-tests/README.md

```diff
@@ -0,0 +1,20 @@
+Snippet-tests
+=====================
+
+Automated tests for the Snippets web app
+
+Running Tests
+-------------
+
+* [Install Tox](https://tox.readthedocs.io/en/latest/install.html)
+* Run `tox`
```

Comment:
Add a line on how to run this against a different host to `snippets.allizom.org`